benchmarks/index.js

```diff
@@ -0,0 +1,43 @@
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var connector = require('..');
+var Benchmark = require('benchmark');
+
+var ds = new DataSource(connector, {
+  host: process.env.LB_HOST || '127.0.0.1',
+  port: process.env.LB_PORT || 27017,
+  database: process.env.LB_DB || 'strongloop'
+});
+var Todo = ds.define('Todo', {
+  content: {type: String}
+});
+
+var suite = new Benchmark.Suite;
+suite
+  .on('start', function() {
+    console.log('#', new Date());
+  })
+  .add('Todo.create', {
+    defer: true,
+    fn: function(deferred) {
+      Todo.create({content: 'Buy eggs'}, function() {
+        deferred.resolve();
+      });
+    }
+  })
+  .add('Todo.find', {
+    defer: true,
+    fn: function(deferred) {
+      Todo.find(function() {
+        deferred.resolve();
+      });
+    }
+  })
+  .on('cycle', function(event) {
+    console.log(String(event.target));
+  })
+  .on('complete', function() {
+    console.log();
+    Todo.destroyAll();
+    process.exit();
```

Comment:
@raymondfeng This lib [does not support async setup/teardown](https://github.com/bestiejs/benchmark.js/issues/70). So we have two choices:

1. `process.exit()` to force the script to quit (items in the db are cleaning up properly since its fire and forget anyways)
2. Write a sync version of `destroyAll` in the connector and use that instead.